### PR TITLE
fix: bindings were broke

### PR DESCRIPTION
**Pull request by jakobrosenberg**
Newest version of serialport now includes prebuilt bindings and I've added `fixbind` as a fallback solution.

`fixbind` is the bindings fallback solution from pymakr-vscode abstracted into a generalized package.

Next step, but low priority, would be to clean up the old tightly coupled binding solution.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,6 +79,7 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.7.0",
     "eslint-plugin-import": "^2.19.1",
+    "fs-extra": "^10.0.0",
```
Comment (elmasria) [inline package.json:82]: seems not used in this changes, lets remove it.

File: package.json [hunk 2]
```diff
@@ -16,8 +16,7 @@
   "repository": "https://github.com/pycom/pymakr-atom",
   "license": "GPL-3.0",
   "engines": {
-    "atom": ">=1.41.0",
-    "node": ">=6.3.0 <=7.0.0"
+    "atom": ">=1.41.0"
```
Comment (elmasria) [inline package.json:19]: I think better to keep the least value of the node, what do you think?

File: lib/main.js [hunk 3]
```diff
@@ -2,13 +2,16 @@
 
 import { CompositeDisposable } from 'atom';
 import Config from './config';
+const { fixbind } = require('fixbind')
```
Comment (elmasria) [inline lib/main.js:5]: fix bindings do not support es6?

File: lib/main.js [hunk 4]
```diff
@@ -2,13 +2,16 @@
...
 
 const $ = require('jquery');
 
 export default {
   config: Config.settings(),
 
   activate(state) {
+
+    fixbind(require, '@serialport/bindings', {/** options */ })
```
Comment (elmasria) [inline lib/main.js:14]: if no option required, maybe better to just pass empty or no value

**Review by elmasria — CHANGES_REQUESTED** [review]
Thank you for the changes I will try to test that, just address a couple of comments

File: package.json (hunk 1 above)
Comment (jakobrosenberg) [reply package.json:82]: `fs-extra` is a peer dependency of fixbind. By default it uses `fs.rmSync`, but for older versions of node, like the one used in Atom, it falls back to `fse.removeSync`.

I can add a @note in package.json if needed.

File: package.json (hunk 2 above)
Comment (jakobrosenberg) [reply package.json:19]: I'm good either way, but we definitely need to get rid of the `<=7.0.0`, unless I'm misunderstanding its meaning.

File: lib/main.js (hunk 3 above)
Comment (jakobrosenberg) [reply lib/main.js:5]: Good catch. It works with both require and import. I can fix it, but IIRC the Atom repo already mixes esm and cjs, so it's just one of many imports that should be corrected.

I'll be happy to convert everything to a single specification. I've already done it for vscode and gotten rid of babel.

**Review by elmasria — APPROVED** [review]

**Review by elmasria — APPROVED** [review]

**Pull request merged**